executor/oci/resolvconf.go

```diff
@@ -74,7 +73,7 @@ func GetResolvConf(ctx context.Context, stateDir string, idmap *idtools.Identity
 
 		if dns != nil {
 			var (
-				dnsNameservers   = resolvconf.GetNameservers(dt, types.IP)
+				dnsNameservers   = resolvconf.GetNameservers(dt, resolvconf.IP)
```

Comment:
Looks like this change wasn't done yet; opened https://github.com/moby/buildkit/pull/2858